.github/workflows/ci-web.yml

```diff
@@ -63,6 +63,11 @@ jobs:
         working-directory: client
         timeout-minutes: 5
 
+      - name: Check prettier
+        run: npx prettier . --check
+        working-directory: client
+        timeout-minutes: 5
```

Comment:
Move that step to be near `check lint` step

Reply:
What do you mean? It's right under the link step.

Reply:
I thought the step wasn't following `check lint`, I should have double check